supported.js

```diff
@@ -1,4 +1,5 @@
 export const extensions = [
+	'avro',
```

Comment:
It was implied that https://github.com/sindresorhus/file-type/pull/597#discussion_r1179941635 applied here too. And the below.